central/graphql/resolvers/loaders/namespaces.go

```diff
@@ -0,0 +1,144 @@
+package loaders
+
+import (
+	"context"
+	"reflect"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/namespace/datastore"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var namespaceLoaderType = reflect.TypeOf(storage.NamespaceMetadata{})
+
+func init() {
+	RegisterTypeFactory(reflect.TypeOf(storage.NamespaceMetadata{}), func() interface{} {
+		return NewNamespaceLoader(datastore.Singleton())
+	})
```

Comment:
May be worth considering changing this to behind the postgres flag, as well as it's usage